presto-hive/src/main/java/com/facebook/presto/hive/authentication/KerberosHiveMetastoreAuthentication.java

```diff
@@ -44,6 +45,28 @@
     public KerberosHiveMetastoreAuthentication(HiveClientConfig hiveClientConfig, @ForHiveMetastore HadoopAuthentication authentication)
     {
         this(hiveClientConfig.getHiveMetastoreServicePrincipal(), authentication);
+        verifyKerberosProperties(hiveClientConfig);
+    }
+
+    private static void verifyKerberosProperties(HiveClientConfig hiveClientConfig)
+    {
+        requireNonNull(hiveClientConfig.getHiveMetastoreServicePrincipal(),
+                "Kerberos principal is not set for metastore service");
```

Comment:
These error messages should give the config property to be set, and be something like "hive.metastore.principal must be set when Kerberos is enabled for Hive."